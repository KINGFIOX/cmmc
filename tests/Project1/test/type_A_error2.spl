int main() {
    int y = 0123;
    float z = 1e-3;
    float w = .0;
    float d = 0.;
    float x = 0x123p+3;
    return 0;
}
