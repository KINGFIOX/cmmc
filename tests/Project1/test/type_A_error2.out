Error type A at Line 2: unknown lexeme 0123
Error type A at Line 3: unknown lexeme 1e
Error type B at Line 1: Missing semicolon ';'
Error type B at Line 4: Missing specifier
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 5: Missing specifier
Error type B at Line 6: Missing semicolon ';'
Error type A at Line 6: unknown lexeme 0x123p
Error type B at Line 6: Missing specifier
Error type B at Line 7: Missing specifier
